pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/MetadataAndDictionaryAggregationPlanClusterIntegrationTest.java

```diff
@@ -418,7 +418,6 @@ public void testDictionaryBasedQueries()
     pqlQuery = "SELECT MAX(ArrTime) FROM " + DEFAULT_TABLE_NAME + " where DaysSinceEpoch > 16100";
     response = postQuery(pqlQuery);
     assertTrue(response.get("numEntriesScannedPostFilter").asLong() > 0);
-    assertTrue(response.get("numEntriesScannedInFilter").asLong() > 0);
```

Comment:
This had to be fixed since DaysSinceEpoch is a sorted column and will be using sorted inverted index operator and doc id set. So the number of entries scanned in filter will be 0.

Reply:
Can you change it to  `assertEquals(response.get("numEntriesScannedInFilter").asLong(), 0)`?

Reply:
Done